ci/hlgroups.make

```diff
@@ -0,0 +1,18 @@
+# vim: ft=make
+SHELL = /bin/bash
+
+# Default target to actually run the comparison:
+.PHONY: check
+.INTERMEDIATE: hlgroups deflinks hlgroups.stripped
+
+check: hlgroups.stripped deflinks
+	diff hlgroups.stripped deflinks
+
+hlgroups:
+	grep '\*hl-' ../runtime/doc/*txt | sed -E 's/.*:<?\s*//' | sed 's/ /\n/g' | sed 's/hl-//' | sed 's/\*//g' | sort > hlgroups
```

Comment:
This seems a bit ugly. Perhaps use a separate sed script like this:
```
#!/bin/sh

sed -ne '/\*hl-/{
s/^<\?\s*//
s/\*//g
s/hl-//g
s/ /\n/
p
}' ../runtime/doc/*.txt |sort
```

or put all the sed invocations into a single command using several `-e` args.

Reply:
> or put all the sed invocations into a single command using several `-e` args.

I ended up doing exactly that, thanks for the hint!

Reply:
> Do we need a clean rule?

I added the intermediate files in the .INTERMEDIATE Makefile target, so a clean rule is not needed any more.